### DISCUSSION TITLE
Windows 11 Python 3.12.5 EASY Newbie Question

### DISCUSSION THREAD

**Discussion by mkmtrading** [beetbox/beets#5496] [Q&A] [answered]
I can’t figure out how to set the default editor to “C:\Program Files\Notepad++\notepad++.exe”

When I run:
C:\Users\mike>beet config -e
error: Could not edit configuration: [Errno 2] No such file or directory

I get a similar error running edit plug-in:
beet edit

The edit plug-in says “Your text editor (i.e., the command in your $VISUAL or $EDITOR environment variable)” but for the life of me I cannot find where to set these environment variables.

I’ve also installed PyCharm Community since I’m assuming I’ll want to use it for $VISUAL

FWIW beets is AWESOME - I have successfully imported most of my music:
beet stats
Tracks: 82358
Total time: 31.3 weeks
Approximate total size: 364.7 GiB
Artists: 11276
Albums: 5022
Album artists: 1878

**Comment by arogl** [comment, marked as answer]
In your DOS prompt try 

`set EDITOR="C:\Program Files\Notepad++\notepad++.exe"`

If that works add it to a `.bat` or `.cmd` file that you can call before you start beets.

In `POWERSHELL/PWSH` 

edit $profile I use VSCODE so i type `code $profile`

add the following line

`$env:EDITOR = &'C:\Program Files\Notepad++\notepad++.exe'`

To work around spaces in the filepath, you need the special `&` style. I might have it wrong so google is your friend for `POWERSHELL/PWSH`

**Comment by mkmtrading** [comment]
That worked, thank you
